### PR TITLE
📝 add troubleshooting to faq

### PR DESCRIPTION
**Pull request by basarat**

### CODE REVIEW

**Comment by basarat** [comment]
Build failure unrelated to this doc only PR :rose:

File: README.md [hunk 1]
```diff
@@ -237,4 +237,7 @@ You may find that Puppeteer does not behave as expected when controlling pages t
 * Puppeteer is bundled with Chromium--not Chrome--and so by default, it inherits all of [Chromium's media-related limitations](https://www.chromium.org/audio-video). This means that Puppeteer does not support licensed formats such as AAC or H.264. (However, it is possible to force Puppeteer to use a separately-installed version Chrome instead of Chromium via the [`executablePath` option to `puppeteer.launch`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#puppeteerlaunchoptions). You should only use this configuration if you need an official release of Chrome that supports these media formats.)
 * Since Puppeteer (in all configurations) controls a desktop version of Chromium/Chrome, features that are only supported by the mobile version of Chrome are not supported. This means that Puppeteer [does not support HTTP Live Streaming (HLS)](https://caniuse.com/#feat=http-live-streaming).
 
+#### Q: I am having trouble installing / running Puppeter on my test enviroment?
```
Comment (ebidel) [inline README.md]: Puppeter -> Puppeteer
no my test enviroment  -> in my test environment

**Review by ebidel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (basarat) [reply README.md]: done :rose:

Comment (ebidel) [reply README.md]: You didn't fix the 2 typos.

Comment (basarat) [reply README.md]: Fixed typo :rose:

Comment (ebidel) [reply README.md]: "environment" is still misspelled

Comment (basarat) [reply README.md]: > "environment" is still misspelled

fixed :rose:

**Review by aslushnikov — APPROVED** [review]

**Pull request merged**